devel/addon/kyverno/kustomization.yaml

```diff
@@ -1,2 +1,17 @@
+# This Kustomization is used to adapt the upstream Pod security policy for use
+# specifically in the cert-manager namespace.
+# Changes ClusterPolicy resources to namespaced Policy.
+# Use as follows:
+#  kustomize build . > policy.yaml
 bases:
   - https://github.com/kyverno/policies/pod-security
+patches:
+  - patch: |-
+      - op: replace
+        path: /kind
+        value: Policy
+      - op: add
+        path: /metadata/namespace
+        value: cert-manager
```

Comment:
Do we now need to install cert-manager before kyverno?
The setup script now fails for me locally when installing kyverno as it doesn't find `cert-manager` ns

Reply:
See https://github.com/jetstack/cert-manager/compare/6089866a2440d41e65c612a65c663bf9be9800d7..6873aa73e83d22f9aaec828d665ec8c9d1318194